aerogear/data-sync-ui
# Saving a resolver for the second time doesn't work

## Motivation
https://issues.jboss.org/browse/AEROGEAR-7754

## What
Fixes second save glitch after selecting a resolver.

## How
`schemaId` is passed by onClick button, but it's overriden to `undefined` after `UpsertQuery` is called. The equivalent field for the Resolver model is `GraphQLSchemaId` so this prop is now used after refetch:
```javascript
const schemaId = resolver.schemaId // defined when resolver does not exist (selected for creation)
                 || resolver.GraphQLSchemaId; // defined after resolver is created
```

## Verification Steps

1. After saving a schema and creating a Data Source, go to Resolvers and select an uncreated resolver.
2. Fill in the forms and click on **Save**.
3. Edit any form and click again on **Save**, changes should be then saved.
